src/QMCWaveFunctions/SplineAdoptorReaderP.h

```diff
@@ -357,7 +357,7 @@ struct SplineAdoptorReader: public BsplineReaderBase
       foundit &= h5f.read(cG,s);
       get_psi_g(ti,spin,cur_bands[iorb].BandIndex,cG);//bcast cG
       double total_norm = compute_norm(cG);
-      if(std::abs(total_norm-1.0)>PW_COEFF_NORM_TOLERANCE)
+      if((std::abs(total_norm-1.0)>PW_COEFF_NORM_TOLERANCE)&(checkNorm))
```

Comment:
In an if-statement && is preferred. Please also move the checkNorm before the check. Please adjust the others also